docs/trouble-shooting.md

```diff
@@ -403,7 +403,7 @@ Notice that the zone's delegated dataset has no more space.
 
 1. Increase the zone's ZFS dataset. In the GZ, run:
 >```
-># zfs set quota=1G zones/e3ab01c5-d5d0-423c-97de-2c71183302d9
+># zfs set quota=100G zones/e3ab01c5-d5d0-423c-97de-2c71183302d9
```

Comment:
I think the number used here is less important than the fact that it's at least higher than the current value. That's implied in the previous statement, but perhaps it needs to be made more clear.

Reply:
I added a few words trying to make it more clear.